src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/IdentifierOfTypeQueryGenerator.cs

```diff
@@ -0,0 +1,57 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics;
+using Microsoft.Health.Fhir.Core.Features.Search.Expressions;
+using Microsoft.Health.Fhir.SqlServer.Features.Schema.Model;
+using Microsoft.Health.SqlServer.Features.Schema.Model;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Search.Expressions.Visitors.QueryGenerators
+{
+    internal class IdentifierOfTypeQueryGenerator : SearchParamTableExpressionQueryGenerator
```

Comment:
So that I understand, why don't we need the `VisitMissingField()` method override that we have in `TokenQueryGenerator.cs`?

Reply:
missing is another modifier, and I don't think you can have multiple modifiers in same time.